framework/behaviors/CacheableBehavior.php

```diff
@@ -0,0 +1,188 @@
+<?php
+
+namespace yii\behaviors;
+
+use yii\base\Behavior;
+use yii\base\Widget;
+use yii\base\WidgetEvent;
+use yii\caching\Cache;
+use yii\caching\Dependency;
+use yii\di\Instance;
+use yii\web\View;
+
+/**
+ * Cacheable widget behavior.
+ *
+ * @property Widget $owner
+ *
+ * @group behaviors
+ */
+class CacheableBehavior extends Behavior
```

Comment:
That's a fair argument; not sure consistency is more important than understandability.

> We don't put traffic lights on each intersection for the sake of consistency.